### DISCUSSION TITLE
Feature request: unified transition specification

### DISCUSSION THREAD

**Discussion by dabrahams** [hakimel/reveal.js#3062] [Ideas]
Looking at your example:
```html
<section data-transition="slide">
    and on …
</section>
<section data-transition="slide-in fade-out">
    and stops.
</section>
<section data-transition="fade-in slide-out">
    (Passengers entering and leaving)
</section>
<section data-transition="slide">
    And it starts again.
</section>
```

One can see that the usual desire is to specify one flavor of transition in-and-out at the boundary between slides, and one speed for the previous transition-out and the next transition-in as well.  It would be great to be able to dispense with `<section>` tags and do something like:

```html
    and on …
<transition data-transition="slide"/>
    and stops.
<transition data-transition="fade"/>
    (Passengers entering and leaving)
<transition data-transition="slide"/>
    And it starts again.
```

I'm by no means an HTML expert, so this may not be quite the best way to express the idiom, but it just seems like in the most common cases, transitions in and out are a logical unit in the author's mind.

**Comment by alinant87** [comment]
Hi
- [ ]
